Obsidian.API/Commands/ArgumentParsers/LocationArgumentParser.cs

```diff
@@ -1,19 +1,24 @@
-﻿using Obsidian.Entities;
+﻿namespace Obsidian.API.Commands.ArgumentParsers;
 
-namespace Obsidian.Commands.Parsers;
-
-public class LocationTypeParser : BaseArgumentParser<VectorF>
+public sealed class LocationArgumentParser : BaseArgumentParser<VectorF>
 {
-    public LocationTypeParser() : base(10, "minecraft:vec3")
+    public LocationArgumentParser() : base(10, "minecraft:vec3")
     {
     }
 
     public override bool TryParseArgument(string input, CommandContext context, out VectorF result)
     {
         var splitted = input.Split(' ');
         var location = new VectorF();
+        var player = context.Player;
+
+        if(player is null)
```

Comment:
Shouldn't this _only_ return when a relative position is tried to be fetched?

When the position is not relative, we don't really necessarily need the player right?

Reply:
Yeah you're right 😅